modules/azure/src/main/java/org/testcontainers/containers/AzuriteContainer.java

```diff
@@ -0,0 +1,529 @@
+            }
+        }
+        exposeRelevantPorts();
+    }
+
+    /**
+     * Returns the connection string for the default credentials.
+     *
+     * @return connection string
+     */
+    public String getDefaultConnectionString() {
+        return getConnectionString(WELL_KNOWN_ACCOUNT_NAME, WELL_KNOWN_ACCOUNT_KEY);
+    }
+
+    /**
+     * Returns the connection string for the account name and key specified.
+     *
+     * @param accountName The name of the account
+     * @param accountKey  The account key
+     * @return connection string
+     */
+    public String getConnectionString(final String accountName, final String accountKey) {
+        return new AzuriteConnectionStringBuilder()
+            .accountCredentials(accountName, accountKey)
+            .useSsl(cert != null)
+            .blobEndpoint(blobHost, getPortNumberFor(blobPort), useBlob)
+            .queueEndpoint(queueHost, getPortNumberFor(queuePort), useQueue)
+            .tableEndpoint(tableHost, getPortNumberFor(tablePort), useTable)
+            .build();
+    }
+
+    String getCommandLine() {
+        final StringBuilder args = new StringBuilder(getExecutableName());
+        if (useBlob) {
+            args.append(" --blobHost ").append(blobHost).append(" --blobPort ").append(blobPort);
+        }
+        if (useQueue) {
+            args.append(" --queueHost ").append(queueHost).append(" --queuePort ").append(queuePort);
+        }
+        if (useTable) {
+            args.append(" --tableHost ").append(tableHost).append(" --tablePort ").append(tablePort);
+        }
+        if (silent) {
+            args.append(" --silent");
+        }
+        if (version) {
+            args.append(" --version");
+        }
+        if (debug != null) {
+            args.append(" --debug ").append(debug);
+        }
+        if (loose) {
+            args.append(" --loose");
+        }
+        if (disableProductStyleUrl) {
+            args.append(" --disableProductStyleUrl");
+        }
+        if (skipApiVersionCheck) {
+            args.append(" --skipApiVersionCheck");
+        }
+        if (inMemoryPersistence) {
+            args.append(" --inMemoryPersistence");
+            if (extentMemoryLimit > NO_LIMIT) {
+                args.append(" --extentMemoryLimit ").append(extentMemoryLimit);
+            }
+        } else {
+            args.append(" --location ").append(location);
+        }
+        if (cert != null) {
+            args.append(" --cert ").append("/cert").append(certExtension);
+            if (pwd != null) {
+                args.append(" --pwd ").append(pwd);
+            } else {
+                args.append(" --key ").append("/key.pem");
+            }
+            if (oauth != null) {
+                args.append(" --oauth ").append(oauth);
+            }
+        }
+        final String cmd = args.toString();
+        logger().debug("Using command line: '{}'", cmd);
+        return cmd;
+    }
```

Comment:
Relate to 

> As I said in the discussion, this implementation must be straightforward. For now, only expose ports would be enough.

Testcontainers doesn't intent to map every command line to the API.

Reply:
Thanks, I will remove the ones with not much value.

Reply:
Done, thank you!